teacher.rb

```diff
@@ -0,0 +1,15 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, specialization)
+    super(age)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Please make sure to also pass `name` and `parent_permission` in the initializer and pass in the super method, So users can set these values as well for teachers.

```ruby
def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```